CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 # Changelog
 
+### 🎁 New Features
+
+* A new `FullScreenSupport` component for providing a child component with full screen expansion
+  capabilities
```

Comment:
could sell this a bit harder.  also worth mentioning the enhancements to various hoist components (3!) to use it, allowing seamless, state-preserving, zoom capabilities

Reply:
Fair enough!